src/main/resources/application.yml

```diff
@@ -0,0 +1,13 @@
+server:
+  port: 8080
+
+management:
+  endpoints:
+    web:
+      base-path: /
+      path-mapping.health: healthcheck
+      exposure:
+        include: info, health, metrics
+  health:
+    mongo:
+      enabled: true
```

Comment:
I think it's a new one. We can use another healthcheck integration like:
 ` health:
    mongo:
      enabled: true
    db:
      enabled: true
    rabbit:
      enabled: true
    redis:
      enabled: true
`